docs/experiments/interpreting-experiments/lift-estimates-and-confidence-intervals/statistical-nitty-gritty.md

```diff
@@ -0,0 +1,560 @@
+and $m_{\tiny{Z}}$, which are normally distributed around the true values with
+variances $\frac{\sigma^2_{\tiny{Y}}}{n}$ and $\frac{\sigma^2_{\tiny{Z}}}{n}$, which
+in turn can be estimated by the sample variances $s_{\tiny{Y}}$ and $s_{\tiny{Z}}$.
+
+Estimating the ratio $\mu$ then becomes a matter of dividing the distributions
+for each component of the ratio; under some reasonable assumptions[^normality],
+the resulting quotient is also approximately normally distributed:
+
+$$
+\frac{\sum_i{\bold{Y}}}{\sum_i{\bold{Z}}} \xrightarrow{D}
+\mathcal{N}\big(\mu_{ratio},~\sigma^2_{ratio}\big)
+$$
+
+The variance term can be calculated
+using the [delta method](https://en.wikipedia.org/wiki/Delta_method)
+(with $\sigma_{\!\tiny{\textit{YZ}}}$ representing the covariance between $\bold{Y}$ and $\bold{Z}$)
+as:
+
+$$
+\sigma^2_{ratio} = \left(
+  \frac{\mu_{\tiny{Y}}}{\mu_{\tiny{Z}}}
+\right)^2
+\left(
+  \frac{\sigma_{\tiny{Y}}^2}{\mu_{\tiny{Y}}^2} + \frac{\sigma_{\tiny{Z}}^2}{\mu_{\tiny{Z}}^2} - \frac{2\sigma_{\!\tiny{\textit{YZ}}}}{n \mu_{\tiny{Y}} \mu_{\tiny{Z}}}
+\right)
+$$
+
+Note that the extra $n$ in the denominator of the last term is because we need the covariance
+between the sample averages.
+
+For each variation, then, we can plug in the sample moments of the numerator and
+denominator metrics to calculate the values for the ratio:
+
+$$
+
+\begin{align}
+m_{ratio} &= \frac{m_{\tiny{Y}}}{m_{\tiny{Z}}} 
+\\
+s^2_{ratio} &= \left(
+  \frac{m_{\tiny{Y}}}{m_{\tiny{Z}}}
+\right)^2
+\left(
+  \frac{s_{\tiny{Y}}^2}{m_{\tiny{Y}}^2} + \frac{s_{\tiny{Z}}^2}{m_{\tiny{Z}}^2} - \frac{2s_{\!\tiny{\textit{YZ}}}}{n m_{\tiny{Y}} m_{\tiny{Z}}}
+\right)
+\end{align}
+$$
+
+The below analysis then simply uses these values for ratio metrics instead of
+the simple sample means and variances for each variation.
+
+:::
+
+### Frequentist analysis
+
+We want to calculate the lift, which we'll call $\Delta$:
+
+$$
+\Delta = \frac{\mu_T - \mu_C}{\mu_C} = \frac{\mu_T}{\mu_C} - 1
+$$
+
+But, since we don't know the true values $\mu_T$ and <NoBreak>$\mu_C$</NoBreak>,
+we'll need to instead *estimate* the lift. We know from the CLT, as shown in
+equation 3 above, that $m_T$ and $m_C$ are approximately normally
+distributed (for sufficiently large $n_C$ and $n_T$); furthermore, since $m_T$
+and $m_C$ are independent, under reasonable assumptions the ratio
+$\frac{m_T}{m_C}$ is approximately normal.[^normality2] This allows us to model
+$\hat{\Delta}$ as a normal distribution:
+
+$$
+\begin{align}
+\hat{\Delta} \sim&~~ \mathcal{N}(\hat{\mu}_{\tiny{\Delta}}, \hat{\sigma}^2_{\tiny{\Delta}})
+\\[1em]
+\hat{\mu}_{\tiny{\Delta}} =&~~ \frac{m_T}{m_C} - 1
+\\[1em]
+\hat{\sigma}^2_{\tiny{\Delta}} =&~~ \frac{m_T^2}{m_C^2}\left(
+  \frac{s^2_C}{m_C^2} + \frac{s^2_T}{m_T^2}
+\right)
+\end{align}
+$$
+
+Note that the calculation of the variance $\hat{\sigma}^2_{\tiny{\Delta}}$ relies on the
+[delta method](https://en.wikipedia.org/wiki/Delta_method).
+
```

Comment:
@bkfunk TODO: add note about minimum requirements here when that's done, and that we rely on big N for all these normality approx.